integration-tests/utils/integration-helpers.js

```diff
@@ -211,13 +281,17 @@ const viewWorkspaceDashboard = async (page, token, workspaceName) => {
   await dismissNotifications(page);
   await fillIn(page, input({ placeholder: 'Search by keyword' }), workspaceName);
   await noSpinnersAfter(page, { action: () => click(page, clickable({ textContains: workspaceName })) });
+
+  // TODO [IA-4682] fix race condition that causes infinite spinner on Analyses page without this delay
+  await delay(Millis.ofSeconds(5));
```

Comment:
Can you add a parameter to this method about whether you need the delay? I would prefer not to have a 5-second delay whenever the workspace dashboard is viewed.

Reply:
Actually, can you just move the delay into `gotoAnalysisTab` and leave this method alone? That makes it much clearer about why/where it is needed and avoids changing the general helper method.

Reply:
Sure; can move to the analysis goto